iothub_client/tests/iothubclient_fuzz_mqtt/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#this is CMakeLists.txt for iothubclient_fuzz_mqtt
+
+if(NOT ${use_mqtt} AND NOT ${use_amqp} AND NOT ${use_http})
+    message(FATAL_ERROR "iothubclient_fuzz_mqtt being generated without protocol support")
+endif()
+
+compileAsC99()
+
+set(iothub_c_files
+    iothubclient_fuzz.c
+)
+
+IF(WIN32)
+    #windows needs this define
+    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+ENDIF(WIN32)
+
+include_directories(.)
+
+add_executable(iothubclient_fuzz_mqtt ${iothub_c_files})
```

Comment:
nit: the executable is called `_mqtt` even if the other transports are used. Should this be called something more neutral?